pkg/collect/host_block_device.go

```diff
@@ -44,22 +45,43 @@ func (c *CollectHostBlockDevices) IsExcluded() (bool, error) {
 }
 
 func (c *CollectHostBlockDevices) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
-	var devices []BlockDeviceInfo
-
 	cmd := exec.Command("lsblk", "--noheadings", "--bytes", "--pairs", "-o", lsblkColumns)
 	stdout, err := cmd.Output()
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to execute lsblk")
 	}
-	buf := bytes.NewBuffer(stdout)
+
+	devices, err := parseLsblkOutput(stdout)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to parse block device output")
+	}
+
+	b, err := json.Marshal(devices)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to marshal block device info")
+	}
+
+	output := NewResult()
+	output.SaveResult(c.BundlePath, HostBlockDevicesPath, bytes.NewBuffer(b))
+
+	return map[string][]byte{
+		HostBlockDevicesPath: b,
+	}, nil
+}
+
+func parseLsblkOutput(output []byte) ([]BlockDeviceInfo, error) {
+	var devices []BlockDeviceInfo
+
+	buf := bytes.NewBuffer(output)
 	scanner := bufio.NewScanner(buf)
 
 	for scanner.Scan() {
 		bdi := BlockDeviceInfo{}
 		var ro int
 		var rm int
+		line := strings.ReplaceAll(scanner.Text(), "MAJ_MIN", "MAJ:MIN")
```

Comment:
I'm assuming some version of `lsblk` introduced `MAJ_MIN` header. Correct? My google-foo isn't good enough

Reply:
Yes. I have added a test where I have included output from both ubuntu 20 and rhel 9 where I encountered the new header https://github.com/replicatedhq/troubleshoot/pull/1045/files#diff-ecdf7d93ef35a4a91a81f07053db659865d964ac0dad0dca6b38de9ec5889852